cli/command_benchmark_compression.go

```diff
@@ -155,11 +212,86 @@ func (c *commandBenchmarkCompression) run(ctx context.Context) error {
 			return compressedSize
 		}
 
+		outputBuffers := makeOutputBuffers(c.parallel, defaultCompressedDataByMethod)
```

Comment:
What prevents GC from running while benchmarking and affecting the results? Is GC turned off? would the last method to be benchmarked be likely to experience more memory pressure?

Reply:
we actually call `runtime.GC()` before each algorithm.